core/quantity/src/Formatter/Formatter.ts

```diff
@@ -231,6 +232,15 @@ export class Formatter {
         const componentText = Formatter.formatCompositePart(unitValue, true, currentLabel, spec);
         compositeStrings.push(componentText);
       }
+
+      if (spec.format.type === FormatType.Ratio){
+        if ( spec.persistenceUnit.phenomenon !== "Units.SLOPE")
```

Comment:
We should not hard code slope here, also we should not ignore the additional composite parts if ratio is selected. Instead, either throw an error if there is more than one part in the composite (saying we require a single part) or format each part as a ratio. Not sure if the second makes sense, we could clarify with Kevin